web/src/components/organisms/Project/ModelsMenu/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useParams } from "react-router-dom";
+
+import ModelsList from "@reearth-cms/components/molecules/Model/ModelsList/ModelsList";
+import ModelCreationModal from "@reearth-cms/components/molecules/Schema/ModelCreationModal";
+
+import useHooks from "./hooks";
+
+export interface Props {
+  title: string;
+  selectModel: (modelId: string) => void;
+}
+
+const ModelsMenu: React.FC<Props> = ({ title, selectModel }) => {
+  const { projectId, modelId } = useParams();
+
+  const {
+    handleModelModalClose,
+    handleModelModalOpen,
+    modelModalShown,
+    handleModelCreate,
+    handleModelKeyCheck,
+    isKeyAvailable,
+    models,
+    model,
+  } = useHooks({
+    projectId,
+    modelId,
+  });
+
+  return (
+    <>
+      <ModelsList
+        title={title}
+        selectModel={selectModel}
+        defaultSelectedKeys={[model?.id ?? ""]}
+        models={models}
+        handleModalOpen={handleModelModalOpen}
+      />
+      <ModelCreationModal
+        isKeyAvailable={isKeyAvailable}
+        projectId={projectId}
+        handleModelKeyCheck={handleModelKeyCheck}
```

Comment:
onModelKeyCheck

Reply:
Changed and fixed. (btw it's a part of an old code that I just moved to another place).